bandit/plugins/django_xss.py

```diff
@@ -149,12 +134,12 @@ def evaluate_call(call, parent, ignore_nodes=None):
     if isinstance(call, ast.Call) and isinstance(call.func, ast.Attribute):
         if isinstance(call.func.value, ast.Str) and call.func.attr == 'format':
             evaluate = True
-            if call.keywords or (six.PY2 and call.kwargs):
+            if call.keywords or call.kwargs:
                 evaluate = False  # TODO(??) get support for this
 
     if evaluate:
         args = list(call.args)
-        if six.PY2 and call.starargs and isinstance(call.starargs,
+        if call.starargs and isinstance(call.starargs,
```

Comment:
I believe starargs only exists in the py2 AST.

Reply:
good catch, in that case I think we can remove this whole line?

```
        if call.starargs and isinstance(call.starargs,
                                                    (ast.List, ast.Tuple)):
            args.extend(call.starargs.elts)
```

Reply:
Yep, think so.